### ISSUE TITLE
glasskube webbundle is missing

### ISSUE THREAD

**Issue by pmig** [Homebrew/homebrew-core#168623]
### `brew gist-logs <formula>` link OR `brew config` AND `brew doctor` output

```shell
https://gist.github.com/pmig/e39724d79b60ad17ac6f71eb8a92abe1
```

### Verification

- [X] My `brew doctor` output says `Your system is ready to brew.` and am still able to reproduce my issue.
- [X] I ran `brew update` and am still able to reproduce my issue.
- [X] I have resolved all warnings from `brew doctor` and that did not fix my problem.
- [X] I searched for recent similar issues at https://github.com/Homebrew/homebrew-core/issues?q=is%3Aissue and found no duplicates.

### What were you trying to do (and why)?

Using the `glasskube serve` command after building the application

### What happened (include all command output)?

The Glasskube webbundle is missing, so no css files are present in the binary, see:

![image](https://github.com/Homebrew/homebrew-core/assets/3041752/e6606055-e764-42e9-8d95-7f5ca774dbbf)

### What did you expect to happen?

Before building the go application `make web` needs to be execute, so our javascript and css bundles are present in `internal/web/root/static/bundle`.

As an alternative it would be possible to copy to webbundles from the release similar to the [NixOS glasskube package](https://github.com/NixOS/nixpkgs/blob/master/pkgs/by-name/gl/glasskube/package.nix).

As another alternative it would be possible to just use the binary, as we do it in the [glasskube tap](https://github.com/glasskube/homebrew-tap/blob/main/glasskube.rb).

### Step-by-step reproduction instructions (by running `brew` commands)

```shell
1. brew install glasskube
2. glasskube serve
3. see frontend without css
```

**Comment by SMillerDev** [comment]
> Before building the go application make web needs to be execute, so our javascript and css bundles are present in internal/web/root/static/bundle.

Could you make a pull request to that extend? It might be easier for someone who has in depth knowledge of the build like you seem to.

**Comment by pmig** [comment]
What would be the preferred method?

1.  run `make web` during build (requires npm)
2. extract / download the frontend bundles from the release before go build
3. use the binaries we already provide similar to our formular

**Comment by SMillerDev** [comment]
I think `make web` is best, extracting is okay. But precompiled binaries are not acceptable in Homebrew core

**Comment by branchvincent** [comment]
Fixed with #168746
